src/containers/QuoteDisplayerFn.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react'; 
+import Button from '../components/Button/Button.js';
+import Quote from '../components/Quote/Quote.js';
+import { fetchQuote } from '../services/fetchQuote.js';
+
+
+const QuoteDisplayerFn = () => {
+  const [quoteObj, onClick] = useState({
+    character: '', 
+    quote: '',
+    image: ''
+  });
+
+  const handleClick = () => {
```

Comment:
I feel like this would be more easily understood upon a quick glance:
```
 const handleClick = () => {
    return fetchQuote()
      .then(quote => setQuoteObj({ quoteObj: quote }));
```